content/about.tsx

```diff
@@ -0,0 +1,50 @@
+import { Icons } from "../components/Icons";
+
+/**
+Markdown supported for description
+*/
+
+export const ABOUT_CONTENT = {
+  WHO_WE_ARE: {
+    TITLE: "Who we are",
+    DESCRIPTION: `TLSNotary is an open-source protocol developed by the **Privacy and Scaling Exploration (PSE)** research lab of the Ethereum Foundation.
```

Comment:
`**...**` is rendered as **<ins>underline</ins>**?

Reply:
yes, where markdown is supported it's indicated in a comment on top of the document

Reply:
In all the md versions I know, `**` is used for **bold**

Reply:
oh, you are right fixed!